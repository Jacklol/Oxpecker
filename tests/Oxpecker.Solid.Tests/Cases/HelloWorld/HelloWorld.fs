module Oxpecker.Solid.Tests.Cases.HelloWorld

open Browser

let helloWorld () =
    console.log "Hello, World!"
