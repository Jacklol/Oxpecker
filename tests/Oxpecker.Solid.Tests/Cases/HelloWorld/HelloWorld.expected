import { some } from "./fable_modules/fable-library-js.4.21.0/Option.js";

export function helloWorld() {
    console.log(some("Hello, World!"));
}

